connect/models/fulfillment.py

```diff
@@ -13,7 +13,17 @@
 
 
 class Fulfillment(BaseModel):
-    pass
+    @property
+    def new_items(self):
+        return [i for i in self.asset.items if i.quantity > 0 and i.old_quantity == 0]
+
+    @property
+    def changed_items(self):
+        return [i for i in self.asset.items if i.quantity > 0 and i.old_quantity > 0]
+
+    @property
+    def removed_items(self):
+        return [i for i in self.asset.items if i.quantity == 0 and i.old_quantity > 0]
 
```

Comment:
Use `filter`
`filter(lambda item: item. quantity > 0 and item.old_quantity == 0, self.asset.items)`

Reply:
Done.